pkg/services/ngalert/store/alert_rule.go

```diff
@@ -416,23 +416,71 @@ func (st DBstore) GetNamespaceByUID(ctx context.Context, uid string, orgID int64
 	return folder, nil
 }
 
+func (st DBstore) getFilterByOrgsString() string {
+	if len(st.Cfg.DisabledOrgs) == 0 {
+		return ""
+	}
+	builder := strings.Builder{}
+	builder.WriteString("org_id NOT IN(")
+	idx := len(st.Cfg.DisabledOrgs)
+	for orgId := range st.Cfg.DisabledOrgs {
+		builder.WriteString(strconv.FormatInt(orgId, 10))
```

Comment:
No prepared statement?

Reply:
Good point! I changed the function to produce safer filter